src/cli/export/export.go

```diff
@@ -37,3 +55,119 @@ func exportCmd() *cobra.Command {
 func handleExportCmd(cmd *cobra.Command, args []string) error {
 	return cmd.Help()
 }
+
+func runExport(
+	ctx context.Context,
+	cmd *cobra.Command,
+	args []string,
+	ueco utils.ExportCfgOpts,
+	sel selectors.Selector,
+	backupID, serviceName string,
+) error {
+	r, _, _, _, err := utils.GetAccountAndConnect(ctx, sel.PathService(), repo.S3Overrides(cmd))
+	if err != nil {
+		return Only(ctx, err)
+	}
+
+	defer utils.CloseRepo(ctx, r)
+
+	exportLocation := args[0]
+	if exportLocation == "" {
+		// This should not be possible, but adding it just in case.
+		exportLocation = control.DefaultRestoreLocation + dttm.FormatNow(dttm.HumanReadableDriveItem)
+	}
+
+	Infof(ctx, "Exporting to folder %s", exportLocation)
```

Comment:
I was mimicking the behaviour for export. I think it would be better to keep them the same.
https://github.com/alcionai/corso/blob/a973f69d182c161d6c30ad3376f43335f20b7c41/src/cli/utils/restore_config.go#L77